### PR TITLE
docs(samples): add/remove bucket labels sample

### PR DESCRIPTION
**Pull request by laljikanjareeya**
PR contains below samples:

- Add bucket labels(storage_add_bucket_label)
- Remove bucket labels(storage_remove_bucket_label)

### CODE REVIEW

File: samples/removeBucketLabel.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//   description: Removes bucket label.
+//   usage: node removeBucketLabel.js <BUCKET_NAME>
+
+function main(bucketName = 'my-bucket', label = 'label') {
+  // [START storage_remove_bucket_label]
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const label= = 'Name of a label, e.g. label';
```
Comment (AVaksman) [inline samples/removeBucketLabel.js:28]:
```suggestion
  // const label = 'Name of a label, e.g. label';
```

File: samples/addBucketLabel.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: Storage Add Bucket Label.
+//   description: Adds bucket label.
+//   usage: node addBucketLabel.js <BUCKET_NAME>
+
+function main(bucketName = 'my-bucket') {
```
Comment (AVaksman) [inline samples/addBucketLabel.js:22]: should we add `label` param here as well?

File: samples/system-test/buckets.test.js [hunk 3]
```diff
@@ -112,6 +112,20 @@ it("should disable a bucket's uniform bucket-level access", async () => {
   );
 });
 
+it('should add label to bucket', async () => {
+  const output = execSync(`node addBucketLabel.js ${bucketName}`);
```
Comment (AVaksman) [inline samples/system-test/buckets.test.js:116]: Should we pass a label here as well?

**Review by AVaksman — CHANGES_REQUESTED** [review]

**Review by AVaksman — APPROVED** [review]

**Comment by frankyn** [comment]
Hi @laljikanjareeya,

Thanks for sending these PRs. We are pending @JesseLovelace return to review these samples. Please continue sending them and we can wait to merge until they're back to review.

cc: @JustinBeckwith for vis. I'll be adding "Do Not Merge" label until then, but this is prioritized work that you can continue @laljikanjareeya.

Thank you.

**Comment by laljikanjareeya** [comment]
Gentle ping @JesseLovelace

**Comment by laljikanjareeya** [comment]
@JesseLovelace Can you take a look into samples PR as this has been since long. Thanks!
cc: @frankyn

File: samples/removeBucketLabel.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
```
Comment (frankyn) [inline samples/removeBucketLabel.js:28]: Double `=`
Does this represent the label_key? Can it be a list of keys?

**Review by frankyn — CHANGES_REQUESTED** [review]

**Review by frankyn — APPROVED** [review]

**Pull request merged**